extensions/apple-notes/src/errors.tsx

```diff
@@ -0,0 +1,39 @@
+import { ActionPanel, Detail, Icon, Action } from "@raycast/api";
+
+export class PermissionError extends Error {
+  constructor(message: string) {
+    super(message);
+    this.name = "PermissionError";
+  }
+}
+
+export const isPermissionError = (error: unknown) => {
+  return error instanceof Error && error.name === "PermissionError";
+};
+
+// @TODO: This screen should be handled by Raycast itself (https://github.com/raycast/extensions/issues/101)
+const permissionErrorMarkdown = `## Raycast needs full disk access in order to display your Apple Notes.
+
+![Full Disk Access Preferences Pane](https://i.imgur.com/3SAUwrx.png)
```

Comment:
The `full-disk-access` image is in the repo.  Not sure if you intended to use it here.

Reply:
Ah you're right!